kb/testing/testing-ci.md

```diff
@@ -0,0 +1,107 @@
+# Mbed TLS CI
+
+All code that is included in Mbed TLS must be contributed in the form of [[https://github.com/Mbed-TLS/mbedtls/pulls | pull requests on GitHub]] and undergoes some automated testing. This page describes the continuous integration jobs that run on every pull request.
+
+## DCO
+
+This job checks that all commits have a `Signed-off-by:` line. The presence of this line indicates that the author of the commit certifies that the commit is covered by the [[https://github.com/Mbed-TLS/mbedtls/blob/development/dco.txt | Developer Certificate of Origin]] and contributed according to the [[https://github.com/Mbed-TLS/mbedtls/blob/development/README.md#License | project license]].
```

Comment:
I'd not come across [[url | text]] for links in markdown before - we seem to normally use [text](url) - and that form is used later in this document, too. Do we care enough about consistency to change one or the other?

Reply:
Oh, that's phabricator syntax - I updated most but obviously missed some. I'll fix it.